comparison.md

```diff
@@ -8,19 +8,19 @@ When you're starting out with modding, it's important to use a template. Templat
 
 ## Migrating from other Templates
-The main difference between the excsh and mirin template is the way that eases work. In the exsch template, eases are kind of like level 2 of modding, where level 1 is mod strings. But in the mirin template, eases are level 1. Eases are simpler to comprehend in mirin because it requires way fewer arguments. You only need to provide the end percent, and the start percent is inferred. That's because the mirin template will remember what value you've eased to last, and then start the next ease from the right percent automatically. Basically, the main point is that I've made eases as simple as possible, so that you can use only eases instead of mixing modstrings and eases. Firstly, this makes it simpler to use the template. You'll never be like "I kind of want this to be eased, but I'm too lazy to do it" ever again. Secondly, because it's using eases everywhere, there's a bunch of stuff that becomes possible that wouldn't be possible in the exsch template. Basically, poptions/nodes/overlapping eases/the reset function/get/default values/etc. is built off of the idea that eases are the core building block used to describe how mods change over time.
+The main difference between the excsh and mirin template is the way eases work. In the exsch template, eases are kind of like level 2 of modding, where level 1 is mod strings. But in the mirin template, eases are level 1. Eases are simpler to comprehend in mirin because it requires way fewer arguments. You only need to provide the end percent, and the start percent is inferred. That's because the mirin template will remember what value you've eased to last, and then start the next ease from the right percent automatically. Basically, the main point is that I've made eases as simple as possible, so that you can use only eases instead of mixing modstrings and eases. Firstly, this makes it simpler to use the template. You'll never be like "I kind of want this to be eased, but I'm too lazy to do it" ever again. Secondly, because it's using eases everywhere, there's a bunch of stuff that becomes possible that wouldn't be possible in the exsch template. Basically, poptions/nodes/overlapping eases/the "reset" function/get/default values/etc. is built off of the idea that eases are the core building block used to describe how mods change over time.
```

Comment:
I like this change! If you wanted to go further with rewording this section, one easy edit would be to take out the word "Basically" and "really" everywhere.

Reply:
true